static/src/js/containers/DatepickerContainer/__tests__/DatepickerContainer.test.js

```diff
@@ -183,11 +294,15 @@ describe('DatepickerContainer component', () => {
 
       test('the "end" input passes the correct object', () => {
         const { enzymeWrapper, props } = setup()
-        const { onChange } = enzymeWrapper.instance()
+
+        enzymeWrapper.instance().picker.current = {}
+        enzymeWrapper.instance().picker.current.setState = jest.fn()
+
+        const { onInputBlur } = enzymeWrapper.instance()
         const { onSubmit } = props
         enzymeWrapper.setProps({ type: 'end' })
 
-        onChange(moment.utc('1988-09-03'))
+        onInputBlur({ target: { value: moment.utc('1988-09-03') } })
```

Comment:
I dont see why this wouldve needed to be swapped to onInputBlur. Arent we still ultimately calling onChange with the values?